add global datasets (#138)
OSGeo/grass-website: content/download/data.en.md

<hr>

### <a name="GlobalDatasets"></a> Global datasets

> Based on amongst others the <a href="https://www.naturalearthdata.com/">Natural Earth data collection</a>, a GRASS GIS location with global

Reviewer: ```suggestion
Based on the <a href="https://www.naturalearthdata.com/">Natural Earth data collection</a>, a GRASS GIS location with global
```

> background maps is available in the World Geodetic System 1984 (WGS84). It is provided in two versions exist, one smaller, basic version and 

Reviewer: ```suggestion
background maps is available in the World Geodetic System 1984 (WGS84). It is provided in two versions, a smaller, basic one and 
```

> a as more comprehensive collection with additional hydrological and ecological data: 

Reviewer: ```suggestion
a more comprehensive collection with additional hydrological and ecological data: 
```

<ul id="links" class="list-unstyled version">
 <li>
  <span class="mwl"><a href="https://zenodo.org/record/3762852#.XrBt0vmxXIE" target="_blank"> Basic global dataset </a></span>
  <a href="https://zenodo.org/record/3762852/files/natural-earth-dataset.zip?download=1" class="inl btn btn-primary" target="_blank">Download ZIP (117 MB)</a>
  </li>
</ul>

<ul id="links" class="list-unstyled version">
 <li>
  <span class="mwl"><a href="https://zenodo.org/record/3359632#.XrBvnPmxXIE" target="_blank"> Full global dataset </a></span>
  <a href="https://zenodo.org/record/3359632/files/global-dataset.zip?download=1" class="inl btn btn-primary" target="_blank">Download ZIP (1.3 GB)</a>
  </li>
</ul>

> On the [Global datasets](https://grasswiki.osgeo.org/wiki/Global_datasets) pages of the GRASS GIS WIKI, the community shares information on other open datasets

Reviewer: ```suggestion
In the [Global datasets](https://grasswiki.osgeo.org/wiki/Global_datasets) wiki, the community shares information on other open datasets
```
